### PR TITLE
Use oid type column to store object ids so vacuumlo does not delete these objects

### LINKED ISSUE #9: Using oid type in table where oid references are stored.
Considering pg_largeobject returns an OID, it will make sense that ID to be stored in the OID type.
Otherwise garbage collection processes like `vacuumlo` cannot count references of orphaned objects.

### PR DESCRIPTION
**Pull request by krists**
Fixes #9

### CODE REVIEW

**Comment by krists** [comment]
@kvlou Could you review these changes?

File: lib/refile/postgres/backend.rb [hunk 1]
```diff
@@ -101,8 +101,14 @@ def registry_table
         if exists?(id)
           with_connection do |connection|
             ensure_in_transaction(connection) do
-              connection.lo_unlink(id.to_s.to_i)
-              connection.exec_params("DELETE FROM #{registry_table} WHERE id = $1::integer;", [id])
+              rez = connection.exec_params(%{
+                SELECT * FROM #{registry_table}
+                WHERE #{registry_table}.oid = $1::integer
```
Comment (kvlou) [inline lib/refile/postgres/backend.rb:106]: any reason why this isn't `$1::oid` ?

File: spec/refile/postgres/backend_spec.rb [hunk 2]
```diff
@@ -64,6 +64,18 @@
     end
   end
 
+  describe "Orphaned large object cleaning" do
```
Comment (kvlou) [inline spec/refile/postgres/backend_spec.rb:67]: a test showing that other unreferenced large objects are vacuumed would be handy. but nbd.

**Comment by kvlou** [comment]
@krists sorry for the late response, looks good and thank you for fixing this. 

i left couple of comments, but otherwise great.

**Pull request merged**